core/src/main/java/io/questdb/cutlass/http/HttpHeaderParser.java

```diff
@@ -356,12 +379,18 @@ private void cookieParse(long lo, long hi) {
         long p0 = lo;
         for (long p = lo; p < hi; p++) {
             char c = (char) Unsafe.getUnsafe().getByte(p);
-            switch (c) {
+            switch (c | 32) {
                 case '=':
                     if (p0 == p) {
                         LOG.error().$("cookie name is missing").$();
                         return;
                     }
+                    if (cookie != null) {
```

Comment:
This approach to unknown attribute seems to mean that any attribute name that ends with a legal name but starts with garbage, such as `xDomain`, passes as the correct name (`Domain`). I changed one of the tests that calla `assertCookieVanilla` to use `xDomain`, and it still passed.

Reply:
ok, i pushed this specific change to ignore only unknown attribute, not the cookie itself https://github.com/questdb/questdb/pull/5142/commits/82b8e961c55840c97a072f0509ca2319909e1f55

Reply:
that particular branch you highlighted kicks in only when there is unknown attribute with a value, e.g. `zt=123`. `z` is important, because it is not from the known alphabet

Reply:
I'm sorry, I didn't want to be specific on that line of code; there are various places the problem could be fixed. Here's the issue -- this is the base case, it passes:

```
assertCookieVanilla(
        "GET /ok?x=a&y=b+c&z=123 HTTP/1.1\r\n" +
                "Set-Cookie: id=123; Domain=hello.com; Path=/; Secure; Partitioned; HttpOnly; Max-Age=1234545; SameSite=strict; Expires=Wed, 21 Oct 2015 07:28:00 GMT\r\n" +
                "\r\n"
);
```

Now I'm adding `x` to spell out `Dxomain`, and it fails:

```
assertCookieVanilla(
        "GET /ok?x=a&y=b+c&z=123 HTTP/1.1\r\n" +
                "Set-Cookie: id=123; Dxomain=hello.com; Path=/; Secure; Partitioned; HttpOnly; Max-Age=1234545; SameSite=strict; Expires=Wed, 21 Oct 2015 07:28:00 GMT\r\n" +
                "\r\n"
);
```

But if I add the `x` like this: `xDomain`, then it passes:

```
assertCookieVanilla(
        "GET /ok?x=a&y=b+c&z=123 HTTP/1.1\r\n" +
                "Set-Cookie: id=123; xDomain=hello.com; Path=/; Secure; Partitioned; HttpOnly; Max-Age=1234545; SameSite=strict; Expires=Wed, 21 Oct 2015 07:28:00 GMT\r\n" +
                "\r\n"
);
```

Reply:
oh, this is a good fuzz, thanks Marko. It is a class of problems i have not tested for!

`DomainX` is recognized as "unknow", so assert on domain name fails
`xDomain`  is recognized as "Domain" so there is a pass. It just ignored prefix, that makes attribute invalid!

Reply:
ok, fixed in https://github.com/questdb/questdb/pull/5142/commits/e9427c3ceee6c980aef327273031519db163b249